iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -594,7 +594,7 @@ void LinalgTileAndFusePass::runOnOperation() {
       });
     }
 
-    if (options.enableVectorization) {
+    if (launchConfig.useVectorize()) {
```

Comment:
It would be nice to kill the cl option entirely and make vectorization on by default. Any remaining work for that?

Reply:
Good point, I think it should all work at this point and we can enable it by default. I'll send another PR for that.